README.md

```diff
@@ -21,8 +21,8 @@ $ helm repo add kedacore https://kedacore.github.io/charts
 $ helm search repo kedacore
 NAME                                            CHART VERSION   APP VERSION     DESCRIPTION
 kedacore/external-scaler-azure-cosmos-db        0.1.0           0.1.0           Event-based autoscaler for Azure Cosmos DB chan...
-kedacore/keda                                   2.14.1          2.14.0          Event-based autoscaler for workloads on Kubernetes
-kedacore/keda-add-ons-http                      0.7.0           0.7.0           Event-based autoscaler for HTTP workloads on Ku...
+kedacore/keda                                   2.14.2          2.14.0          Event-based autoscaler for workloads on Kubernetes
+kedacore/keda-add-ons-http                      0.8.0           0.8.0           Event-based autoscaler for HTTP workloads on Ku...
```

Comment:
I forgot this change during the add-on release, it's not related with this directly but it doesn't hurt if we upgrade it here either xD